packages/notifications/src/styled/StyledNotification.ts

```diff
@@ -5,36 +5,84 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, getColor } from '@zendeskgarden/react-theming';
 import { VALIDATION_TYPE, ARRAY_VALIDATION_TYPE } from '../utils/types';
+import { StyledTitle } from './content/StyledTitle';
+import { StyledClose } from './content/StyledClose';
+import { StyledBase } from './StyledBase';
 
 const COMPONENT_ID = 'notifications.notification';
 
 export interface IStyledNotificationProps {
   type?: VALIDATION_TYPE;
 }
 
+const closeStyles = (props: ThemeProps<DefaultTheme>) => {
+  const { theme } = props;
+  const { colors, palette } = theme;
+  const { neutralHue } = colors;
+
+  return css`
+    ${StyledClose} {
+      color: ${palette[neutralHue][600]};
+    }
+
+    ${StyledClose}:focus {
+      outline: none;
+      background-color: ${getColor(neutralHue, 600, theme, 0.15)};
+      color: ${palette[neutralHue][800]};
```

Comment:
These should not be applied to the `:focus` pseudo. Leverage `[data-garden-focusvisible]` instead.